website/static/style.css

```
@@ -21,71 +22,92 @@ body {
     flex-direction: column;
 }
 
-#title, a#title:link, a#title:visited, a#title:hover, a#title:active {
+#title, a#title:link, a#title:visited, a#title, a#title:active {
     font-weight: 1000;
     font-size: x-large;
-    color: black;
+    color: white;
     text-decoration: none;
 }
-#subtitle, a#subtitle:link, a#subtitle:visited, a#subtitle:hover, a#subtitle:active {
-    color: black;
+#subtitle, a#subtitle:link, a#subtitle:visited, a#subtitle, a#subtitle:active {
+    color: white;
     text-decoration: none;
-    font-weight: 700;
+    font-weight: 300;
+}
+
+.tip {
+    font-style: italic;
+    font-size: 0.7em;
+    color: var(--light-text-color)
 }
 
 .grid-item{
     grid-row-gap: 3px;
 }
 
-.grid-item-title {
-    word-wrap: break-word;
-    font-weight: 700;
-}
 .grid-item-title > a {
+    display: inline-block;
     text-decoration: none;
+    font-weight: 400;
 }
+
 .grid-item-title > a:link {
     color: var(--main-color);
 }
 .grid-item-title > a:visited {
     color: var(--main-color);
-    opacity: 0.70;
+    opacity: 0.7;
 }
 
 .details, .details a:link, .details a:visited {
-    color: #696969;
+    color: var(--light-text-color);
     font-size: x-small;
     margin-top: 3px;
 }
 
-.grid-item-datepub {
+.content, container {
+    display: grid;
+    margin: 5px;
+}
+
+.content {
+    grid-row-gap: 10px;
 }
 
 .container {
-    margin: 5px;
-    /*padding-right: 15px;*/
     display: grid;
     grid-row-gap: 20px;
-    /*justify-content: center;*/
 }
 
-
+a:hover {
+    text-decoration: underline !important;
+}
 
 .navigation {
-    text-align: center;
-    font-weight: 600;
-    margin-top: 10px;
-    color: var(--main-color);
+    text-align: right;
+    margin: 10px;
 }
 
-.navigation>a {
-    border: solid;
-    border-radius: 10px;
-    border-width: 2px;
-    padding: 3px;
-    border-color: var(--main-color);
+.navigation > a {
+    border-radius: 20px;
+    padding: 8px 20px;
+    background-color: var(--light-bg-color);
     text-decoration: none;
-    color: var(--main-color);
+    color: inherit;
+}
+
+.navigation > a:hover {
+    text-decoration: none !important;
+    opacity: 0.7;
+}
+
+.navigation > a#prev::before {
+    content: "←";
```

Les fleches c'est une super idée, je trouve que celles ci rendent bien
```suggestion
    content: "↩";
```